.github/workflows/sync-addon-metadata-translations.yml

```diff
@@ -55,3 +55,4 @@ jobs:
           branch: amt-sync
           delete-branch: true
           path: ./project
+          reviewers: gade01
```

Comment:
Thanks for your PR!
Is `reviewers: gade01` intended?

Reply:
Yes, gade01 reviews all the translations